index.html

```diff
@@ -219,5 +219,27 @@ <h2>Skills</h2>
         </ul>
       </div>
       </section> 
+
+      <section class="contact-section">
+        <div class="contact-head">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</div>
+        <div class="form">
+            <input class="form-input" type="text" name="name" placeholder="Full name">
+            <input class="form-input" type="email" name="email" placeholder="Email address">
+            <textarea name="mail-message" id="" cols="20" rows="5" placeholder="Enter text here"></textarea>
+            <input class="send" type="submit" value="Get in touch">
```

Comment:
- [ ] As stated 😃 by the previous reviewer kindly Name input (can be first name and last name field depending on the template used) has maxlength="30" attribute.
Message input has maxlength="500" attribute.💯
All input fields have the required attribute to activate the validation on submission.

Reply:
Does that mean that the name needs to be divided into 2 inputs: first and last name?